kratos/python/add_processes_to_python.cpp

```diff
@@ -552,6 +552,23 @@ void  AddProcessesToPython(pybind11::module& m)
     .def(py::init<Model&, Parameters>())
     ;
 
+    py::class_<FindIntersectedGeometricalObjectsProcess<Element, Condition>, FindIntersectedGeometricalObjectsProcess<Element, Condition>::Pointer, Process>(m,"FindIntersectedElementsConditionsProcess")
```

Comment:
As I told you before, I wouldn't export this to Python since it is not needed in any place. Besides, it makes no sense at all to call this process from Python, since the it only sets the Octree and compute the intersections accordingly, which are useless if called as a standalone utility.